Sources/AWSLambdaRuntimeCore/LambdaHandler.swift

```diff
@@ -135,9 +154,18 @@ extension EventLoopLambdaHandler where Output == Void {
 
 /// An `EventLoopFuture` based processing protocol for a Lambda that takes a `ByteBuffer` and returns a `ByteBuffer?` asynchronously.
 ///
-/// - note: This is a low level protocol designed to power the higher level `EventLoopLambdaHandler` and `LambdaHandler` based APIs.
+/// - note: This is a low level protocol designed to power the higher level ``EventLoopLambdaHandler`` and
+///         ``LambdaHandler`` based APIs.
 ///         Most users are not expected to use this protocol.
 public protocol ByteBufferLambdaHandler {
+    /// Create your Lambda handler for the runtime.
+    ///
+    /// Use this to initialize all your resources that you want to cache between invocations. This could be database
+    /// connections and HTTP clients for example. It is encouraged to use the given `EventLoop`'s conformance
+    /// to `EventLoopGroup` when initializing NIO dependencies. This will improve overall performance, as it
+    /// minimizes thread hopping.
+    static func factory(context: Lambda.InitializationContext) -> EventLoopFuture<Self>
```

Comment:
maybe `makeHandler`? factory is generally not popular term in Swift. cc @weissi for ideas

Reply:
Yeah, generally "make" is the word to use; `makeHandler` would work too.